app/javascript/src/assets/scss/views/_signup.scss

```diff
@@ -1,25 +1,25 @@
 .signup {
   // margin-top: $space-larger*1.2;
 
-  .signup__hero {
+  .signup-hero {
```

Comment:
We follow BEM, so `.signup--hero` should work here.

Reply:
Okay. I was reviewing other files and saw that some places are used with a single hyphen. Was trying get a convention there.

Should I also apply this change to classes like `signup-box`, `accept-terms` aswell?

Reply:
Yes, please

Reply:
Okay, done!